app/templates/volunteer/volunteerProfile.html

```diff
@@ -0,0 +1,58 @@
+{% extends "base.html" %}
+
+{% block scripts %}
+{{super()}}
+  <script type="text/javascript" src="{{url_for('static', filename='js/volunteerIndicateInterest.js') }}?u={{lastStaticUpdate}}"></script>
+
+{% endblock %}
+
+{% block app_content %}
+<div class="container">
+  <div class="row">
+    <table class="table">
+
+      {% if upcomingEvents|length > 0 %}
+        <thead class="tableHeader">
+          <th scope="col"><h4><strong>Program</strong></h4>
+          <th scope="col"><h4><strong>Event Name</strong></h4>
+          <th scope="col"><h4><strong>Start Date</strong></h4>
+          <th scope="col"><h4><strong>End Date</strong></h4>
+        </thead>
+        {% for event in upcomingEvents %}
+        <tr>
+          <td>{{event.singleProgram.programName if event.singleProgram else '--'}}</td>
+          <td>{{event.eventName}}</td>
+          <td>{{event.startDate}}</td>
+          <td>{{event.endDate}}</td>
+        </tr>
+        {% endfor %}
+      {% else %}
+        <h2> Please select the programs you would like to receive updates on. <h2>
```

Comment:
I'd say the text should be more like "Choose interests below to be notified about upcoming events."